### DISCUSSION TITLE
Issue accessing smallstep with traefik

### DISCUSSION THREAD

**Discussion by tardich** [smallstep/certificates#776] [Q&A]
I have configured smallstep on my kubernetes cluster with the kubernetes instructions. Now, I need to configure traefik to get certificates from it. I have configured a new certificatesresolver in traefik (configured the storage and all, a bit like I've configured LetsEncrypt in another instance). Now, when I'm looking at the traefik logs, I see:

`cannot get ACME client get directory at blahblahblah urn:ietf:params:acme:error:accountDoesNotExist :: Account does not exist.`

What is traefik talking about, and how do I instruct traefik about this account?

Christian...

**Comment by tashian** [comment]
Hi Christian, you may be encountering something similar to #749, where the CA is returning a 404, and Traefik's ACME client interprets that as "account does not exist."

The `caServer` setting for Traefik should be `https://ca.example.com:port/acme/{your ACME provisioner name}/directory`.
